verification/curator-service/api/src/controllers/users.ts

```diff
@@ -44,3 +44,33 @@ export const list = async (req: Request, res: Response): Promise<void> => {
         return;
     }
 };
+
+/**
+ * Update a single user's roles.
+ */
+export const updateRoles = async (req: Request, res: Response): Promise<void> => {
+    try {
+        const user = await User.findOneAndUpdate(
+            { googleID: req.params.googleID },
```

Comment:
We should identify users with their mongo ID, not their google ID really, (the _primary key_ index is on the id field, there's no index on googleID).

Reply:
Done.